dbbackup/db/base.py

```diff
@@ -141,8 +143,13 @@ def run_command(self, command, stdin=None, env=None):
         full_env.update(self.env)
         full_env.update(env or {})
         try:
-            process = Popen(cmd, stdin=stdin, stdout=stdout, stderr=stderr,
-                            env=full_env)
+            if type(stdin) == ContentFile:
```

Comment:
Could you change `type(stdin) == ContentFile` to `isinstance(stdin, ContentFile)` ?